src/action.ts

```diff
@@ -45,19 +54,25 @@ export async function createAnIssue (tools: Toolkit) {
       tools.exit.failure(err)
     }
     if (existingIssue) {
-      try {
-        const issue = await tools.github.issues.update({
-          ...tools.context.repo,
-          issue_number: existingIssue.number,
-          body: templated.body
-        })
-        setOutputs(tools, issue)
-        tools.exit.success(`Updated issue ${issue.data.title}#${issue.data.number}: ${issue.data.html_url}`)
-      } catch (err) {
-        tools.exit.failure(err)
+      if (! updateExisting) {
```

Comment:
Extra space here - but for the clarity of the three different options, how do you feel about changing it to:

```suggestion
      if (updateExisting === false) {
```

Normally I'd frown at that kind of verbosity, but because the three states are so explicit I think this is clearer.